relayer/relay/tests/config/fixtures/simple_config.toml

```diff
@@ -0,0 +1,43 @@
+# This is an IBC relayer sample configuration
```

Comment:
There doesn't seem to be large differences between this file and the (already existing) [relayer-conf-example.toml](https://github.com/informalsystems/ibc-rs/blob/86fb4087c5c7c077eb3413127a442312afe29548/relayer/relay/tests/config/fixtures/relayer_conf_example.toml)? The repo would look better if we'd keep just 1 example config file.

Reply:
`relayer_conf_example.toml` is used by the test https://github.com/informalsystems/ibc-rs/blob/fa4cf48a9d3d2e675a94c2ee6dd5e2c9963676b1/relayer/relay/src/config.rs#L142
It is more complex than the `simple_config.toml` as it covers also the different ways to specify channels/ paths, its purpose being to validate and load the configuration file. This said, I believe we need to comb through the `.toml` syntax we use as there is a lot of room for improvement.

Reply:
simple_config.toml is ready to be used with the current `simd` we have for development. `simd` should go away when we have a valid gaiad working on the SDK that we can query against.

With a valid SDK node, relayer-conf-example.toml is ready to be used with the chain_a and chain_b docker images.

I wouldn't want to merge the two, they are for different use-cases.

Reply:
Maybe we can just write this distinction down somewhere in the repo so it's clear to folks that would come across them what the difference is? Or at least in the comments at the top of the toml files?

Reply:
Good idea, I've added a description line into the config files.

Reply:
Thank you, I was missing this context here!